konch.py

```diff
@@ -954,6 +955,25 @@ def preview_unauthorized() -> None:
     return None
 
 
+def __ensure_global_cfg_loaded(mod: typing.Union[types.ModuleType, None]) -> None:
+    """Load global config if not already loaded."""
+    try:
+        if (
+            globals()["_cfg"] is None
+            or globals()["_cfg"] == {}
+            or globals()["_cfg"] == Config()
+        ):
+            raise UnboundLocalError
```

Comment:
Why raise an UnboundLocalError if you're going to handle it below?

Reply:
Accessing the variable if undefined would raise an UnboundLocalError, so raising the same error if variable is essentially blank simplifies the flow since in either case we want to do the same afterwards.